ui/src/app/shared/search-multi-select/search-multi-select.component.spec.ts

```diff
@@ -52,7 +52,7 @@ describe("SearchMultiSelectComponent", () => {
   it("clear field should clean input control", () => {
     component.inputFc.patchValue("value")
     component.clearField()
-    expect(component.inputFc.value.length).toBe(0)
+    expect(component.inputFc.value?.length).toBe(0)
```

Comment:
Please use the non-null operator ! instead of the optional ? operator. This will ensure "value" is forced to be defined prior to the comparison.
`expect(component.inputFc.value!.length).toBe(0)`
https://blog.logrocket.com/understanding-exclamation-mark-typescript/